examples/employee-app/Dockerfile.native

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
You don't need this file unless you intend to support native image with GraalVM.

Reply:
Deleted.